pages/index.tsx

```diff
@@ -100,7 +110,7 @@ export const getServerSideProps: GetServerSideProps = async ({ req, locale }) =>
     scenarioContent = getScenarioContent(claimData)
   } catch (error) {
     // If an error occurs, log it and show 500.
-    logger.error(error)
+    logger.error(error, 'Application error')
```

Comment:
Note: I added a message string to make it easier to locate these as new messages in Application Insights. We can change this at any time.